fs/fs.go

```diff
@@ -58,12 +60,15 @@ type RealFsInfo struct {
 	labels map[string]string
 
 	dmsetup dmsetupClient
+
+	thinPoolWatcher *volume.ThinPoolWatcher
```

Comment:
I'd prefer not overloading FsInfo for exposing device mapper usage. Can we instead have the fsHandler include this logic?